### PR TITLE
Redesigning the Placed Order Preview Page

### LINKED ISSUE #1867: Payment Review Page Not Clear
The payment review page is actually the same page as the previous page with the only difference of the payment button. Please take inspiration from the final payment page of Amazon and make the last page in the payment process similar to Amazon.

screenshot from 2019-01-17 02-12-12

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This PR redesigns the placed order Preview page along the lines of 

#### Changes proposed in this pull request:

- Phase 1
  - Redesigning the event information component
  - Realignment of Payment tabs and swapping the positions of event information and order summary
  - Elaborating the Organiser details  and relocating it
- Phase 2
  - Breaking down Ticket Holder-information into Payee Info and Attendee Info
  - Removing `input_fields` and using placeholders instead
  - Creation of new ticket-holder component to hold Payee Info
- Phase 3
  - Adding Details Row in the ticket-holder section
  - Removing Payment information from billing section
  - Making Billing Information section visibility toggle enabled

Now : Fix build and work on reviews

Fixes #1867

### CODE REVIEW

**Comment by uds5501** [comment]
After Phase 1 Changes in design:

### Placed order page
image

### Successful order page
image

**Comment by uds5501** [comment]
Changes after Phase 2

### Placed Order Review Page

image

### Successful Order Completion Page
image

@CosmicCoder96 What else should be done in the redesign?

**Comment by CosmicCoder96** [comment]
@uds5501 Looks good, I will review in depth soon.

**Comment by uds5501** [comment]
@CosmicCoder96 A quick doubt, how can I access a component's attribute `enableBIlling` in another component? For the moment I have made the Billing Information toggle-able as an alternative

**Comment by uds5501** [comment]
@CosmicCoder96 Review acknowledged, making the changes

**Comment by uds5501** [comment]
@CosmicCoder96 Please review

File: app/templates/components/orders/event-info.hbs [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+        </div>
+      </div>
+    </div>
+  {{/if}}
   <div class="ui secondary segment">
     <h3 class="weight-400">{{t 'When & Where'}}</h3>
   </div>
   <div class="ui padded segment">
-    {{data.event.locationName}}
+    <b>At {{data.event.locationName}}</b>
```
Comment (CosmicCoder96) [inline app/templates/components/orders/event-info.hbs]: use strong tag.

File: app/templates/components/orders/event-info.hbs [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
     <br>
-    {{header-date data.event.startsAt}} - {{header-date data.event.endsAt}}
+    <b>From :</b> {{header-date data.event.startsAt}}
```
Comment (CosmicCoder96) [inline app/templates/components/orders/event-info.hbs]: use `strong` tag.

File: app/templates/components/orders/event-info.hbs [hunk 3]
```diff
@@ -1,11 +1,25 @@
...
+    <br>
+    <b>To :</b> {{header-date data.event.endsAt}}
```
Comment (CosmicCoder96) [inline app/templates/components/orders/event-info.hbs]: use `strong` tag.

File: app/templates/components/orders/event-info.hbs [hunk 4]
```diff
@@ -1,11 +1,25 @@
...
+    {{#if data.event.organizerName}}
+      <b>Organized By :</b> {{data.event.organizerName}}
```
Comment (CosmicCoder96) [inline app/templates/components/orders/event-info.hbs]: use `strong` tag

File: app/templates/orders/placed.hbs [hunk 5]
```diff
@@ -14,45 +14,46 @@
...
+            amount=paymentAmount
+            key=model.order.event.stripeAuthorization.stripePublishableKey
+            onToken=(action "processStripeToken")
+            onClosed=(action "checkoutClosed")
+            onOpened=(action "checkoutOpened")
+          }}
+        </div>
+      {{/if}}
+      {{#if isPaypal}}
+        <div class='paypal-button' style='float: right'>
```
Comment (CosmicCoder96) [inline app/templates/orders/placed.hbs]: why do you need custom styling for this, use semantic ui classes for right float.

**Comment by uds5501** [comment]
@CosmicCoder96  I am afraid I can't find a way to get these boxes at an equal height. What shall I do for them?

**Pull request merged**